packages/browser-sdk/src/lib/embed/index.ts

```diff
@@ -108,6 +108,7 @@ interface WherebyEmbedElementEventMap {
 
 interface WherebyEmbedElementCommands {
     endMeeting: () => void;
+    leaveMeeting: () => void;
```

Comment:
I'm torn on the naming and whether this should be `leaveRoom` to match the command we are sending. Ultimately this is a small thing so happy to go with `leaveMeeting` if we want that instead for this command.

Reply:
Resolved in latest commit